inst/examples/12_counter/R/my-module.R

```diff
@@ -0,0 +1,23 @@
+mymoduleUI <- function(id, label = "Counter") {
+  ns <- NS(id)
```

Comment:
Add a comment here with something like "All UI identifier values must be namespaced using `id`"